holidays/countries/timor_leste.py

```
@@ -29,18 +29,19 @@ class TimorLeste(
 ):
     """
     References:
-    - https://mj.gov.tl/jornal/lawsTL/RDTL-Law/RDTL-Laws/Law-2005-10.pdf  # 2005 Law
-    - http://timor-leste.gov.tl/?p=14494&lang=en  # 2016 Amendment
-    - http://timor-leste.gov.tl/?p=30266&lang=en  # 2022
-    - http://timor-leste.gov.tl/?p=31750&lang=en  # 2023 (en_US)
-    - http://timor-leste.gov.tl/?p=31750&lang=pt  # 2023 (pt_PT)
-    - http://timor-leste.gov.tl/?p=31750&lang=tp  # 2023 (tet)
-    - http://timor-leste.gov.tl/?p=35833&lang=en  # 2024
-    - https://timor-leste.gov.tl/?p=41492&lang=en # 2025
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)